### PR TITLE
new DE and updated EN

### PR DESCRIPTION
**Pull request by pedall**
- [x] created german translation
- [x] updated 2 strings for english language file

### CODE REVIEW

File: src/languages/de-DE.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const { Language, util } = require('klasa');
+
+module.exports = class extends Language {
+
+	constructor(...args) {
+		super(...args);
+		this.language = {
+			DEFAULT: (key) => `${key} wurde noch nicht für de-DE übersetzt.`,
+			DEFAULT_LANGUAGE: 'Standard Sprache',
+			SETTING_GATEWAY_EXPECTS_GUILD: 'Der Parameter <Guild> erwartet entweder einen Server oder ein Serverobject.',
```
Comment (dada1134) [inline src/languages/de-DE.js]: Obje**k**t

File: src/languages/de-DE.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Fehlende notwendige Auswahl: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Deine Auswahl trifft auf keine der folgenden Möglichkeiten zu: (${possibles})`,
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error) => `${tag} | **${error}** | Sie haben **30** Sekunden, um einen gültigen Wert einzugeben. Mit "ABORT" brechen Sie den Vorgang ab.`,
+			MONITOR_COMMAND_HANDLER_ABORTED: 'Abgebrochen',
+			INHIBITOR_COOLDOWN: (remaining) => `Sie haben gerade erst diesen Befehl benutzt. Der Befehl kann von Ihnen wieder in ${remaining} Sekunden verwendet werden.`,
+			INHIBITOR_DISABLED: 'Dieser Befehl ist aktuell deaktiviert.',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Ungenügende Berechtigungen, es fehlt: **${missing}**`,
+			INHIBITOR_PERMISSIONS: 'Ihnen fehlt die Berechtigung diesen Befehl auszuführen.',
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `Aufgrund der fehlenden Einstellung **${settings.join(', ')}** kann dieser Befehl nicht ausgeführt werden.`,
+			INHIBITOR_RUNIN: (types) => `Dieser Befehl kan nur in ${types} Kanälen ausgeführt werden`,
```
Comment (dada1134) [inline src/languages/de-DE.js]: kan**n**

File: src/languages/de-DE.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+			RESOLVER_INVALID_INT: (name) => `${name} muss eine Ganzzahl sein.`,
+			RESOLVER_INVALID_FLOAT: (name) => `${name} muss eine gültige Zahl sein.`,
+			RESOLVER_INVALID_URL: (name) => `${name} muss eine gültige URL sein.`,
+			RESOLVER_STRING_SUFFIX: ' Buchstaben',
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} muss genau ${min}${suffix} sein.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} muss zwischen ${min} und ${max}${suffix} sein.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} muss größer als ${min}${suffix} sein.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} muss kleiner als ${max}${suffix} sein.`,
+			COMMANDMESSAGE_MISSING: 'Es fehlen ein oder mehrere Argumente am Ende der Eingabe.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} ist ein ist ein zwingend erforderliches Argument.`,
```
Comment (dada1134) [inline src/languages/de-DE.js]: I'd say you don't need `zwingend`
> ${name} ist ein erforderliches Argument.

File: src/languages/de-DE.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Deine Auswahl trifft auf keine der folgenden Möglichkeiten zu: (${possibles})`,
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error) => `${tag} | **${error}** | Sie haben **30** Sekunden, um einen gültigen Wert einzugeben. Mit "ABORT" brechen Sie den Vorgang ab.`,
+			MONITOR_COMMAND_HANDLER_ABORTED: 'Abgebrochen',
+			INHIBITOR_COOLDOWN: (remaining) => `Sie haben gerade erst diesen Befehl benutzt. Der Befehl kann von Ihnen wieder in ${remaining} Sekunden verwendet werden.`,
+			INHIBITOR_DISABLED: 'Dieser Befehl ist aktuell deaktiviert.',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Ungenügende Berechtigungen, es fehlt: **${missing}**`,
+			INHIBITOR_PERMISSIONS: 'Ihnen fehlt die Berechtigung diesen Befehl auszuführen.',
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `Aufgrund der fehlenden Einstellung **${settings.join(', ')}** kann dieser Befehl nicht ausgeführt werden.`,
+			INHIBITOR_RUNIN: (types) => `Dieser Befehl kan nur in ${types} Kanälen ausgeführt werden`,
+			INHIBITOR_RUNIN_NONE: (name) => `Der ${name} Befehl wurde nicht für die Benutzung in mindestens einem Channel aktiviert.`,
```
Comment (dada1134) [inline src/languages/de-DE.js]: Kanal

File: src/languages/de-DE.js (hunk 1 above)
Comment (pedall) [inline src/languages/de-DE.js]: done :+1:

File: src/languages/de-DE.js (hunk 3 above)
Comment (pedall) [inline src/languages/de-DE.js]: done :+1:

File: src/languages/de-DE.js (hunk 2 above)
Comment (pedall) [inline src/languages/de-DE.js]: done :+1:

File: src/languages/de-DE.js (hunk 4 above)
Comment (pedall) [inline src/languages/de-DE.js]: done :+1:

File: src/languages/de-DE.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+			RESOLVER_INVALID_INT: (name) => `${name} muss eine Ganzzahl sein.`,
+			RESOLVER_INVALID_FLOAT: (name) => `${name} muss eine gültige Zahl sein.`,
+			RESOLVER_INVALID_URL: (name) => `${name} muss eine gültige URL sein.`,
+			RESOLVER_STRING_SUFFIX: ' Buchstaben',
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} muss genau ${min}${suffix} sein.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} muss zwischen ${min} und ${max}${suffix} sein.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} muss größer als ${min}${suffix} sein.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} muss kleiner als ${max}${suffix} sein.`,
+			COMMANDMESSAGE_MISSING: 'Es fehlen ein oder mehrere Argumente am Ende der Eingabe.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} ist ein ist ein erforderliches Argument.`,
```
Comment (dada1134) [inline src/languages/de-DE.js]: You missed this. It wasn't clear sorry. 

> ${name} ist ein ~~ist ein~~ erforderliches Argument.

Comment (pedall) [inline src/languages/de-DE.js]: done :+1:

File: src/languages/de-DE.js [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+			COMMAND_CONF_ADDED: (value, key) => `Erfolgreich den Wert \`${value}\` zum Attribut: **${key}** hinzugefügt`,
+			COMMAND_CONF_UPDATED: (key, response) => `Erfolgreiche Aktualisierung des Attributs **${key}**: \`${response}\``,
+			COMMAND_CONF_KEY_NOT_ARRAY: 'Dieses Attribut ist nicht vom Typ "array". Bitte benutzen Sie stattdessen die Aktion \'reset\'.',
+			COMMAND_CONF_REMOVE: (value, key) => `Erfolgreich den Wert \`${value}\` vom Attribut: **${key}** entfernt`,
+			COMMAND_CONF_GET_NOEXT: (key) => `Es scheint so, dass das Attribut **${key}** nicht existiert.`,
+			COMMAND_CONF_GET: (key, value) => `Der hinterlegte Wert des Attributs **${key}** ist: \`${value}\``,
+			COMMAND_CONF_RESET: (key, response) => `Das Attribut **${key}** wurde zurückgesesetzt auf: \`${response}\``
+		};
+	}
+
```
Comment (bdistin) [inline src/languages/de-DE.js]: Trailing space (fix lint)

**Pull request merged**